### PR TITLE
Reland "py2-3: more stage2 fixes (#406). (#472)" (#476)"

### PR DESCRIPTION
**Pull request by oliverchang**
This reverts commit 80dac9abe8ddfcf8464077ca18e300912af5c0a6.

Also apply stage1 and 2 fixes to the new fuchsia files.

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 backports.lzma==0.0.13
+configparser==3.7.4
```
Comment (oliverchang) [inline src/requirements.txt]: This should address the previous breakage.

File: docker/oss-fuzz/host/start_host.py [hunk 2]
```diff
@@ -13,6 +13,11 @@
 # limitations under the License.
 """Start host."""
 from __future__ import print_function
+
+from future import standard_library
+standard_library.install_aliases()
+
```
Comment (inferno-chromium) [inline docker/oss-fuzz/host/start_host.py]: can remove newline here, just like other instances.

File: src/python/bot/startup/run_bot.py [hunk 3]
```diff
@@ -32,6 +32,9 @@
 from python.base import modules
 modules.fix_module_search_paths()
 
+from future import standard_library
+standard_library.install_aliases()
+
```
Comment (inferno-chromium) [inline src/python/bot/startup/run_bot.py]: can remove newline to match others.

**Pull request merged**